src/vm/exceptionhandling.cpp

```diff
@@ -6184,11 +6187,11 @@ bool ExceptionTracker::IsInStackRegionUnwoundBySpecifiedException(CrawlFrame * p
 
     // Remember that sfLowerBound and sfUpperBound are in the "OS format".
     // Refer to the comment for CallerStackFrame for more information.
-#if defined(_TARGET_AMD64_)
+#ifndef USE_FUNCLET_CALL_HELPER
```

Comment:
I don't feel like the fact where we get the current SP is related to the stack range bounds. We get it inside of the asm helper just because we need the value at the caller of the funclet and the caller is the asm helper, while in the other case the caller is directly the CallHandler. Hence the different place we capture it.
It sounds it would be ok though to introduce a new ifdef, something like STACK_RANGE_BOUNDS_ARE_CALLER_SP if you feel it makes the conditions easier to understand.